openvdb_houdini/SOP_OpenVDB_Filter.cc

```diff
@@ -326,6 +326,9 @@ Offset:\n\
 
     // Register this operator.
     hvdb::OpenVDBOpFactory("VDB Filter", SOP_OpenVDB_Filter::factory, parms, *table)
+#ifndef SESI_OPENVDB
+        .setParentName("vdbsmooth")
+#endif
```

Comment:
Is "parent" the right terminology for this? I wouldn't, for example, describe sesis vdbsmooth node as the parent of the aswf's vdb filter node. Is this intended to be used in other ways? If not, maybe sometime more specific would be appropriate e.g. setSESIPairName() / setLinkedNodeName() / setNodeTwinName(). Also, do we actually need the ifdef guards?

Reply:
I'm not particularly happy with the name parent either, though I'm not sure your suggestions are any more intuitive. Perhaps native or upstream?

I think we should use the ifdef guards, because otherwise if SideFX were to build this code, they'd end up making a node be it's own a parent. While we could ensure that there were appropriate guards around ophiding to ensure that logic doesn't get executed for SideFX, I'm not sure it's a good idea because this could be used in other ways in the future.

Regarding the VDB Filter/Smooth, my other PR renamed OpenVDB Filter to VDB Smooth (along with others like OpenVDB Convert -> Convert VDB). I didn't include this change here because I wanted this PR to not alter any existing behavior.

Reply:
Let's go with "Native".  "Parent" is definitely not the right term; there's no hierarchical relationship.

Reply:
The `ifdef` can be moved inside the method body, I think.